index.html

```diff
@@ -112,91 +114,132 @@ <h1 style="margin-top: 10px">Smart figures with metadata</h1>
+
+          The script can be uploaded using two alternative workflows, both of which require you to be an admin.
+        </p>
+        <p>
+          <ul>
+            <li>
+              Option 1: Connect to the OMERO server and upload script via the CLI. It is important to be in the correct directory when uploading so that the script is uploaded with the full path <strong>omero/figure_scripts/Figure_To_Pdf.py</strong>:<br>
+              <code style="width:650px">
+                $ cd omero_figure/scripts <br>
+                $ path/to/OMERO.server/bin/omero script upload omero/figure_scripts/Figure_To_Pdf.py --official
+              </code>
+            </li>
+            <li>
+              <i>Option 2</i>: Alternatively, before starting the OMERO.server, copy the script from the figure install <strong>/omero_figure/scripts/omero/figure_scripts/Figure_To_Pdf.py</strong> to the OMERO.server <strong>path/to/OMERO.server/lib/scripts/omero/figure_scripts</strong>. Then restart the OMERO.server.
+          </li> 
+          </ul>
+          Now install the script's dependencies:
           <ul>
             <li>Install <a href="https://bitbucket.org/rptlab/reportlab">reportlab</a> PDF python package.<br>
-              In the simplest case: <code>$ pip install reportlab</code> OR  <code>$ easy_install reportlab</code><br>
-              NB: For python 2.6, you'll need Reportlab 2.7: <code>$ pip install reportlab==2.7</code>
+              <code style="width:650px">
+                $ pip install reportlab
+              </code>
             </li>
-            <li>Optional: Figure legends can be formatted using Markdown syntax. To see this correctly in the
-              exported PDF info page, we need <a href="https://pythonhosted.org/Markdown/index.html">
-              Python-markdown</a>.
-              In the simplest case: <code>$ pip install markdown</code> OR  <code>$ easy_install markdown</code>
-            <li>Upload figure export script (need to be Admin):
-              <code style="width:700px">
-                $ cd figure/scripts   &nbsp &nbsp &nbsp # NB: important to be in this directory<br>
-                $ path/to/bin/omero script upload omero/figure_scripts/Figure_To_Pdf.py --official
+            <li>Optional: Figure legends can be formatted using Markdown syntax. To see this correctly in the exported PDF info page, we need <a href="https://pythonhosted.org/Markdown/index.html">Python-markdown</a> installed.<br>
+              <code style="width:650px">
+                $ pip install markdown
               </code>
             </li>
           </ul>
         </p>
-
         <h1>Update instructions</h1>
+        <h3>Upgrade the app</h3>
         <p>
+          Stop OMERO.web.
+          To upgrade a version of OMERO.figure prior to <strong>2.0.1</strong>:
```

Comment:
Maybe "upgrade *from* a version..." to distinguish from "upgrade *to* a version..."